projects/control-service/projects/pipelines_control_service/src/main/resources/db/migration/V20213110120000__add_last_execution_info_data_job.sql

```diff
@@ -0,0 +1,4 @@
+alter table data_job
```

Comment:
The script should be idempotent (https://www.cockroachlabs.com/docs/stable/add-column.html).

`alter table if exists ... add column if not exists ...`

Reply:
Will do, thanks!